com/company/Arrays/Arrays.java

```diff
@@ -0,0 +1,16 @@
+package com.company.Arrays;
+
+public class Arrays {
+    public static void main(String[] args) {
+        int y = 10;
+        int x = 10;
+        int[][] array = new int[y][x];
```

Comment:
1. create method that will create matrix -> createMatrix(int rows, int columns), something like that

2. create method that will print matrix of any length printMatrix(......)